### ISSUE TITLE
Unable to pass multiple argument in npm script

### ISSUE THREAD

**Issue by abhishek153** [srs/gradle-node-plugin#303]
I have npm run task defined in package.json script as below

"e2e-remoteServer": "protractor ./protractor.conf.js",

using it in my gradle task as:

task npmRunTest(type: NpmTask) {
  args = ['run','e2e-remote']
}

I want to pass multiple arguments to run task : --baseUrl="http://localhost:8080/, --browser="firebox"

How can i achieve this using this plugin. I tried multiple option as 
args = ['run','e2e-remote', ''arg1, 'arg2'] but it seems to simply ignoring the other args. I am unable to run npm scripts similar to above.

Please help how I can achieve this?

Thanks,
Abhishek

**Comment by david-hm** [comment]
Did you try args=['run', 'e2e-remote', '--baseUrl="http://localhost:8080/', '--browser="firebox"']?

**Comment by thSoft** [comment]
Maybe try `args=['run', 'e2e-remote', '--', '--baseUrl="http://localhost:8080/', '--browser="firebox"']` (note the [--](https://docs.npmjs.com/cli-commands/run-script.html))

**Comment by deepy** [comment]
I don't think this is an issue with the plugin, it seems to be specific for npm.
The plugin just sends the arguments, if npm ignores them it won't matter what you send
